lib/galaxy/webapps/galaxy/api/security.py

```diff
@@ -0,0 +1,20 @@
+import logging
+
+from galaxy.web import (
+    expose_api,
+    require_admin
+)
+from galaxy.webapps.base.controller import (
+    BaseAPIController
+)
+
+log = logging.getLogger(__name__)
+
+
+class SecurityController(BaseAPIController):
+
+    @require_admin
```

Comment:
There is a encode_id in the config controller. Can you put this on that controller instead - and add a matching integration test in test_configuration_decode.py (maybe renaming that file)?

Reply:
> There is a encode_id in the config controller. Can you put this on that controller instead - and add a matching integration test in test_configuration_decode.py (maybe renaming that file)?

do you mean to move it under `galaxy/lib/galaxy/security/` directory?

Reply:
https://github.com/galaxyproject/galaxy/blob/dev/test/integration/test_configuration_decode.py
https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/configuration.py#L100

Reply:
> https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/api/configuration.py#L100

thanks, did know about this method. This helps a lot